docs_src/using_click/tutorial001.py

```diff
@@ -7,7 +7,7 @@
 def hello(count, name):
     """Simple program that greets NAME for a total of COUNT times."""
     for x in range(count):
-        click.echo("Hello %s!" % name)
+        click.echo(f"Hello {name}!")
```

Comment:
While F-strings are typically nicer and more readable, there are a few cases where you might prefer not using them. When we're logging things for instance, you might prefer avoiding the `str()` functionality for cases where the statement wouldn't actually get logged (e.g. if the logging level is higher). That said - I'm not actually sure how `click.echo()` operates under the hood so I'm not sure whether we want this rewrite in this case or not.

Reply:
Ah! This one is just that it was an exact copy of Click's tutorial. But they now updated it to use f-strings, so I would take this change. :nerd_face:

Reply:
cf https://github.com/tiangolo/typer/pull/891